test/addons_test.go

```diff
@@ -24,26 +24,34 @@ func TestAddons(t *testing.T) {
 		t.Fatal(err)
 	}
 
-	b, err := ioutil.ReadFile("../addons/zookeeper/0.x/zookeeper.yaml")
+	addons, err := temp.Addons("../addons/")
 	if err != nil {
 		t.Fatal(err)
 	}
-	ro, err := test.DecodeObjectFromManifest(b)
-	if err != nil {
-		t.Fatal(err)
+
+	// Kafka needs special attention right now to ensure it can find its dependency ZK
```

Comment:
Eventually would we be able to move this into the kafka dir, or make it so we don't have a huge if statement here solving the deps somehow?

Reply:
Yes we have a JIRA open with Kudo team about improvements to the Kafka operator. Zain is working on a feature that will be something like auto-discovery for Kafka to find its ZK.